cssconcat.php

```diff
@@ -55,7 +55,7 @@ function do_items( $handles = false, $group = false ) {
 				$css_url = wp_style_loader_src( $css_url, $obj->handle );
 			}
 
-			$css_url_parsed = parse_url( $obj->src );
+			$css_url_parsed = parse_url( is_string( $obj->src ) ? $obj->src : '' );
```

Comment:
Line 50 suggests that `$obj->src` may be `true`. This approach is safer than an explicit cast to `string`.